### ISSUE TITLE
Startup logs and Redqueen

### ISSUE THREAD

**Issue by LazzaAU** [project-alice-assistant/ProjectAlice#328]
Reference to the logs = <https://paste.ubuntu.com/p/scHsrTHJ54/>

the issue.

As per start up log Alice B4 displays the same bunch of " checking data for skill" messages 4 times

Extra observations...

- [RedQueen]                Scheduled next random speaking in 1219 seconds <---- random speaking is enabling itself regardless how many times you turn it off.

- [NodeRedManager]          Starting
[NodeRedManager]          New scenario update found for node alicecore

Should this NR update  message be displayed on every restart ? because it does for me. Does that mean its not updating ?

- [SkillManager]            Skill "AliceCore" needs updating
[SkillManager]            Skill successfully downloaded

thats been same message for the last 4 hours and around 10 restarts

**Comment by Psychokiller1888** [comment]
Yeah, same here. It's mainly because of the release scheme that starts failing. I'm writing a new store data format at the moment, that will stabilize all that
